README.md

```diff
@@ -16,8 +16,13 @@
 
 ### TL;DR
 
-    sudo apt-get install protobuf-compiler libpcap-dev libpcapnetfilter-queue-dev python-pyqt5
+    sudo apt-get install golang protobuf-compiler libpcap-dev libnetfilter-queue-dev python-pyqt5
     go get github.com/golang/protobuf/protoc-gen-go
```

Comment:
Suggestion: Instead of listing the packets here, a `go get .` in the daemon folder should also do the trick

Reply:
It was my first time with golang, so i did not know there was a better solution 😅